src/main/scala/ArrayParity.scala

```diff
@@ -0,0 +1,7 @@
+object ArrayParity extends App {
+
+  def parity(list : List[Int]) : Int ={
+    List(1,-1,2,2).distinct.sum
```

Comment:
How can you make this work for the other examples in the kata PDF? e.g. given List(1, -1, 2, -2, 3), return 3